pom.xml

```diff
@@ -90,7 +90,6 @@
         <groupId>io.grpc</groupId>
         <artifactId>grpc-netty-shaded</artifactId>
         <version>${grpc-netty-shaded.version}</version>
-        <scope>runtime</scope>
```

Comment:
Need this to use NettyChannelBuilder, as ManagedChannelBuilder in `io.grpc` does not allow modifying its SSL context.